Doc/library/os.rst

```diff
@@ -3946,6 +3946,7 @@ written in Python, such as a mail server's external command delivery program.
 
 
 .. data:: CLD_EXITED
+          CLD_KILLED
```

Comment:
What about CLD_STOPPED?

Please add also the `versionchanged` directive.

Reply:
@serhiy-storchaka Oh thanks for letting me know.